spacy/lang/ru/tokenizer_exceptions.py

```diff
@@ -388,8 +393,9 @@
     {ORTH: "прим.", NORM: "примечание"},
     {ORTH: "прим.ред.", NORM: "примечание редакции"},
     {ORTH: "см. также", NORM: "смотри также"},
-    {ORTH: "кв.м.", NORM: "квадрантный метр"},
-    {ORTH: "м2", NORM: "квадрантный метр"},
+    {ORTH: "см. ", NORM: "смотри"},
```

Comment:
I think the space here is a mistake?

```suggestion
    {ORTH: "см.", NORM: "смотри"},
```